column_index.go

```diff
@@ -364,6 +364,27 @@ func splitFixedLenByteArrayList(size int, data []byte) [][]byte {
 }
 
 func boundaryOrderOf(minOrder, maxOrder int) format.BoundaryOrder {
+	if minOrder == 2 || maxOrder == 2 {
```

Comment:
I've been reading through code for 5 minutes now to try to figure out what "2" means in this context... can we declare a variable with value 2, and then refer to the variable in the function? Or add a comment to this function, or to internal/bits/order.go, or ?

Reply:
Currently, `internal/bits/order` returns `-1, 0, 1` to mean [ordered descending, unordered, and ordered ascending](https://github.com/segmentio/parquet-go/blob/54f6b56fa4371c350c43fbb91c648993f9053157/internal/bits/order_amd64.s#L5-L7). This PR adds `2` to mean `equal`.

Ideally, these would all be constants. The one factor complicating those efforts is the assembly. https://github.com/segmentio/parquet-go/blob/54f6b56fa4371c350c43fbb91c648993f9053157/internal/bits/order_amd64.s